.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,29 +1,34 @@
-#### Code Sample, a copy-pastable example if possible
+---
+name: Bug Report
+about: Create a bug report to help us improve pandas
+title: "BUG:"
+labels: ""
```

Comment:
I've left labels blank for now but we could automatically add relevant labels.  Might also be nice to create a "Needs Triage" label to also add if we want to do this.